docs/reference/cloud/elastic-cloud-kibana-settings.md

```diff
@@ -87,6 +87,15 @@ stack: ga 9.2
 stack: ga 9.1
 ```
 
+`xpack.actions.email.maximum_body_length`
+:    The maximum length of an email body.  Values longer than this length will
+be truncated.  The default is 25MB, the maximum is 25MB.
+
+### Version 9.3+ [ec_version_9_3]
```

Comment:
I think this section should be added [here](https://github.com/elastic/kibana/pull/239572/files#diff-8202d1baa76ac9e22a114563bb7dbb55e2305985e6bfc8a234bc249aa6bb5729L76) to match numerical order that the versions are generally following.